.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Codespell Development Environment",
+    // TODO: Switch to image 'mcr.microsoft.com/devcontainers/python:1-3' once
+    // https://github.com/codespell-project/codespell/issues/2897 has been
+    // resolved.
+    "image": "mcr.microsoft.com/devcontainers/python:1-3-bullseye",
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "github.vscode-github-actions",
```

Comment:
Why is the above extension needed?

Reply:
None of these extensions are strictly needed, but they set default extensions for this dev container in VSCode and Codespaces. This is an official extension for working with GitHub's actions and I've found it pretty useful, but it can definitely be removed and installed manually if the user desires. Really just a matter of personal preference.

Reply:
It's just that [GitHub.vscode-github-actions](https://github.blog/2023-03-28-announcing-the-github-actions-extension-for-vs-code/) seems orthogonal to the basic purpose of this container. GitHub Actions are necessary to modify codespell locally.

Reply:
None of these plugins are strictly needed to modify Codespell, but Codespell does make use of GitHub Actions and this is a recommended plugin for those files. One of the primary benefits of dev containers in my mind is to give a single-button development environment will all the plugins and settings for VSCode that are considered standard for a project already configured.

In my mind, for anyone working on Codespell's actions, this plugin will be appreciated and VSCode will prompt them to install it regardless. And for anyone who isn't, they probably won't even notice it's there.

Reply:
Perhaps just change from
```
github.vscode-github-actions
```
to
```
GitHub.vscode-github-actions
```
to match the [GitHub.vscode-github-actions](https://marketplace.visualstudio.com/items?itemName=github.vscode-github-actions) page.

Reply:
I'm definitely not opposed to the change, but VSCode handles these identified weirdly. I pulled that identifier from the page in VSCode where it's still lower-cased and I've confirmed it's functioning as expected:
<img width="964" alt="image" src="https://github.com/codespell-project/codespell/assets/84342833/36d6c192-7425-49e9-b285-95c3d14b39cd">

I suspect upper-case will likely work, so if you have a strong opinion there, I can definitely test the alternative.

Reply:
Microsoft tools are usually case-insensitive. My idea was to stick to the documentation, but if the documentation itself is inconsistent, let's keep as it is. I prefer lowercase myself.